Suggest explanation: allow some personal data, in accordance to #676 (#691)
datenanfragen/website: content/en/suggest.md

Considering these tips when suggesting companies helps us a lot, but is <strong>completely optional</strong>. Whether you follow none of these tips, one, some, or all: We are happy about every suggestion!

- The first and best starting point for obtaining information about a company is the privacy policy. It also makes for a great `source`.

> - We only allow personal data in our database if its absolutely nesseccary. That means, we only accept e.g. <code>john.doe@example.org</code> as email if no generic privacy-related email like <code>privacy@example.org</code> exists.

Reviewer: ```suggestion
- We only allow personal data in our database if it's absolutely necessary. That means, we only accept e.g. <code>jane.doe@example.org</code> as the email if no generic privacy-related email like <code>privacy@example.org</code> exists.
```

Author: > nesseccary  

wow :D
